Library/Formula/jx_osx.rb

```diff
@@ -0,0 +1,17 @@
+class JxOsx < Formula
+  homepage "http://jxcore.com/"
+  url "https://s3.amazonaws.com/nodejx/jx_osx64.zip"
```

Comment:
This looks like a binary. If it is one it should be submitted to `homebrew-binary` instead.